src/pwr.ps1

```diff
@@ -886,6 +897,17 @@ if ($Run) {
 	Assert-NonEmptyPwrPackages
 	Invoke-PwrScripts
 	exit
+} elseif ($Which) {
+	$local:Pkg = Assert-PwrPackage $Which
+	Write-PwrHost "$($local:Pkg.Ref) $($local:Pkg.Digest)"
+	exit
+} elseif ($Where) {
+	$local:Pkg = Assert-PwrPackage $Where
+	if (-not (Test-PwrPackage $local:Pkg)) {
+		Write-PwrFatal "package $($local:Pkg.Ref) is not installed"
+	}
+	Write-PwrHost (Resolve-PwrPackge $local:Pkg)
```

Comment:
Do we want to use regular write host so the pwr prefix is not in the output? I'm imagining a use where you inline the command as part of another command.

Something like

`PS > Get-ChildItem (pwr where jdk) -File -Recurse -Include '*target file*' `

Reply:
`Write-PwrHost` does not append the `pwr: ` prefix.

Reply:
Ah